portfolio/src/main/webapp/index.html

```diff
@@ -7,31 +7,36 @@
     <script src="script.js"></script>
   
   </head>
-  <body onload="getComments()">
-    <div id="content">
+  <body   onload="getComments()" >
+    <div id="content" >
       <h1>Charles Lovett's Portfolio</h1>
       
       <form action="/Comments" method="POST">
-           
-          
           <p>Post a comment</p>
-          <input type="text" name="comment">
+          <input type="text" name="comment" >
           <br/><br/>
-      <input type="submit" />
-     
+          <input type="submit" />
       </form>
 
       <form action="/Comments" method="POST">
           <p>Maxium comments</p>
-          <input type="number" name="maxnum">
+          <input type="number" name="maxnum" >
           <br/><br/>
-          <input type="submit" />
+          <input type="submit" />  
       </form>
 
       <br/><br/>
       
       <ul id="fetch-comment"></ul>
 
+      <form id="uploaded-image" method="POST" enctype="multipart/form-data" >
```

Comment:
Do you need to set an `action` URL here? (I don't know the answer to this, but I suspect that by default, this just sends a post request to the index page URL.)